crates/local-cluster-runner/src/node/mod.rs

```diff
@@ -56,12 +56,9 @@ pub struct Node {
     #[builder(mutators(
             #[mutator(requires = [base_dir])]
             pub fn with_node_socket(self) {
-                let ts = std::time::SystemTime::now()
-                    .duration_since(std::time::UNIX_EPOCH)
-                    .expect("to get timestamp")
-                    .as_secs();
+                let r: u64 = rand::random();
                 let node_socket: PathBuf = PathBuf::from(self.base_config.node_name()).join("node.sock");
-                let tokio_console_socket: PathBuf = PathBuf::from(self.base_config.node_name()).join(format!("tokio_console-{ts}.sock"));
+                let tokio_console_socket: PathBuf = PathBuf::from(self.base_config.node_name()).join(format!("tokio_console-{r}.sock"));
```

Comment:
If binding to the tokio console socket is a problem, wouldn't it be the same for the `node_socket` as well when restarting a node with the same name?

Reply:
Is the `tokio_console_socket` meant to be used in any way by the user or is it only meant to avoid port conflicts with multiple processes that are executed with the `console` feature?

Reply:
@tillrohrmann It's not a problem for the node_socket because we do this here https://github.com/restatedev/restate/blob/main/crates/core/src/network/net_util.rs#L103

IMHO the tokio_console_socket is not meant to be used by the user in the CI tests and is mainly to avoid port conflicts with the console feature is enabled. Yet if that socket is needed it can always be fetched from the config files.

Reply:
Should we do the same here: https://github.com/restatedev/restate/blob/d596a74c96e3398d34756b330a3e8e1cb0aac91a/crates/tracing-instrumentation/src/lib.rs#L281?